XOR model export in CI

JacobSzwejbka opened the pull request:
Summary: Add model to CI since it got broken recently

Differential Revision: D65768331

malfet commented on extension/training/examples/XOR/test/test_export.py:
```diff
@@ -0,0 +1,24 @@
+
+# pyre-unsafe
+
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
```
License headers are cool, two license headers makes code twice cool?
```suggestion
```

JacobSzwejbka committed:
XOR model export in CI (#6756)

Summary:

Add model to CI since it got broken recently

Reviewed By: malfet, georgehong

Differential Revision: D65768331
